js/5etools-adventures.js

```diff
@@ -11,7 +11,7 @@ function d20plusAdventure () {
 	d20plus.adventures.load = function (url) {
```

Comment:
we could make this `d20plus.adventures.load` function `async` instead, and have it read a bit nicer below, e.g.:

```js
const data = await DataUtil.loadJSON(url);

function isPart (e) {
...
```